linode/manager
# upcoming: [M3-7293] - AGLB Full Create Flow - Service Targets

## Description 📝
- Adds _Service Targets_ section to the AGLB create flow

## Preview 📷
![Screenshot 2023-12-06 at 10 27 44 AM](https://github.com/linode/manager/assets/115251059/ba7b8d27-98b7-4e1e-9c38-1333c4f39cf6)

## How to test 🧪

### Prerequisites
- Have the `AGLB` and `AGLB Full Create Flow` flags **on**

### Verification steps 
- Go to http://localhost:3000/loadbalancers/create 
- Test existing functionality
- Test new Service Targets section
  - Verify you can add a Service Target
  - Verify you can remove a Service Target
  - Verify you can edit a Service Target

## As an Author I have considered 🤔
